code/modules/events/false_alarm.dm

```diff
@@ -7,15 +7,16 @@
 	announceWhen	= 0
 	endWhen	=	90
 	var/datum/event_meta/EM
+	var/eventname
 
 
 /datum/event/false_alarm/end()
```

Comment:
Somewhere here, EM should also be assigned to the GC with qdel, and `EM = null` executed. Obviously, this is provided that this datum has no further use for the EM.